src/SDKs/CognitiveServices/dataPlane/Vision/ContentModerator/ContentModerator.Tests/Helpers/Constants.cs

```diff
@@ -0,0 +1,1296 @@
+		TEXT_WORKFLOW,
+		[Description("BVTVideoWorkflow")]
+		VIDEO_WORKFLOW
+	}
+
+
+
+	public enum StandardReviewRequests
+	{
+		[Description(@"")]
+		CREATE_TEXT_REVIEW,
+		[Description(@"[   {     ""Type"": ""Image"",    ""Content"": ""https://pbs.twimg.com/media/BfopodJCUAAjmkU.jpg:large"",     ""ContentId"": ""ImageReview"",    ""Metadata"": [      {        ""Key"": ""KeyNote1"",        ""Value"": ""KeyNote2""      }    ]  }]")]
+		CREATE_IMAGE_REVIEW,
+		[Description(@"[ {    ""VideoFrames"": [   { ""Id"":""frm1"", ""Timestamp"":0, ""FrameImage"":""http://cvs-docs.azurewebsites.net/test-data/A1.jpg"",""Metadata"":[ {""Key"":""apiScoreVal"",""Value"":""0.206""}, {""Key"":""a"",""Value"":""False""}],""ReviewerResultTags"": [] },   { ""Id"":""frm2"",""Timestamp"":1,""FrameImage"":""http://cvs-docs.azurewebsites.net/test-data/A2.jpg"",""Metadata"":[ {""Key"":""apiScoreVal"",""Value"":""0.206""}, {""Key"":""a"",""Value"":""False""}],""ReviewerResultTags"": []},   { ""Id"":""frm3"",""Timestamp"":4,""FrameImage"":""http://cvs-docs.azurewebsites.net/test-data/A1.jpg"",""Metadata"":[ {""Key"":""apiScoreVal"",""Value"":""0.206""}, {""Key"":""a"",""Value"":""False""}],""ReviewerResultTags"": [] }],
+		""Metadata"":[{""Key"":""apiScoreVal"", ""Value"":0.206},{""Key"":""a"",""Value"":""False"" },{""Key"":""racyScore"",""Value"":""0.151""}, {""Key"":""r"",""Value"":""False""},{""Key"":""ExternalId"",""Value"":""roar.mp4"" } ],""Type"":""Video"",""Content"":""https://rvdevmediaservicetest.streaming.mediaservices.windows.net/f7f073c3-66e5-436d-acee-827d3437df29/Roar.ism/manifest"",""ContentId"":""image02"",""Status"":""UnPublished"",""CallbackEndpoint"":"""", ""TimeScale"":""10""}]")]
+		CREATE_VIDEO_REVIEW,
+		[Description(@"[ 
+			 { 
+					""Timestamp"": ""1289"",
+					""FrameImage"":""http://cvs-docs.azurewebsites.net/test-data/A2.jpg"", 
+					""Metadata"": [ 
+					  { 
+					""Key"": ""apiScoreVal"",
+					""Value"": ""0.206""
+				  } 
+					], 
+					""ReviewerResultTags"": [
+					]
+			}, 
+ 
+				  { 
+					""Timestamp"": ""1389"", 
+					""FrameImage"":""http://cvs-docs.azurewebsites.net/test-data/A2.jpg"", 
+					""Metadata"": [
+					  { 
+					""Key"": ""apiScoreVal"",
+					""Value"": ""0.206"" 
+				  } 
+					], 
+					""ReviewerResultTags"": [
+					] 
+				  } 
+				  ] ")]
+		ADD_VIDEO_Frames,
+		[Description(@"WEBVTT
+
+		00:11.000 --> 00:13.000
+		<v Roger Bingham>We are in New York City
+
+		00:13.000 --> 00:16.000
+		<v Roger Bingham>We're actually at the Lucern Hotel, just down the street
+
+		00:16.000 --> 00:18.000
+		<v Roger Bingham>from the American Museum of Natural History
+
+		00:18.000 --> 00:20.000
+		<v Roger Bingham>And with me is Neil deGrasse Tyson
+
+		00:20.000 --> 00:22.000
+		<v Roger Bingham>Astrophysicist, Director of the Hayden Planetarium")]
+		ADD_VIDEO_TRANSCRIPTS,
+		[Description(@"[
+			{
+				""TimeStamp"": ""123"",
+
+				""Terms"":
+
+				[
+					{
+						""Index"": 0,
+						""Term"" : ""fuck""
```

Comment:
@v-sodsou can you clarify what is this all about?
With this, you will be blocked at sign/publish time when we run bunch of static analysis tools.

Reply:
This was test data from our test team. Modified the test data.